Content.Shared/_CorvaxNext/Clothing/Components/ClothingGrantTagComponent.cs

```diff
@@ -0,0 +1,11 @@
+namespace Content.Shared._CorvaxNext.Clothing;
+
+[RegisterComponent]
+public sealed partial class ClothingGrantTagComponent : Component
+{
+    [DataField("tag", required: true), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField(required: true), ViewVariables(VVAccess.ReadWrite)]
```

Если я не ошибаюсь, ReadWrite по умолчанию стоит в датафилдах.